alarm/src/consumer.ts

```diff
@@ -39,22 +39,20 @@ export const setupConsumer = async (): Promise<void> => {
   await consumer.connect()
   await consumer.subscribe({ topics: await getTopics(), fromBeginning: false })
 
-  // TODO: andranno aggiunte anche le regole inerenti alle camere
-  //console.log(await getSensorRules())
-  //TODO TO TEST MULTIPLE GETS
   await securityRuleService.getExceedingRules()
+  await securityRuleService.getIntrusionRules()
 
   consumer
     .run({
       eachMessage: async ({ topic, message }): Promise<void> => {
-        if (message.key === null || message.value === null) return
-        const messageKey: Buffer = message.key
+        if (message.value === null) return
         const messageValue: Buffer = message.value
 
-        console.log('Message num: ' + JSON.parse(messageKey.toString()))
+        console.log('Arrived message', messageValue.toString())
         const rawValues = JSON.parse(messageValue.toString())
-
         if (topic.startsWith('CAMERA')) {
+          const date = new Date(rawValues.timestamp)
+          console.log('TEST DATE', date)
```

Comment:
we can keep this and after i'll link the securityRuleService to do the active check 👍